extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/AstroHandlerFactory.java

```diff
@@ -21,28 +23,29 @@
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 
-import com.google.common.collect.Sets;
-
 /**
  * The {@link AstroHandlerFactory} is responsible for creating things and thing handlers.
  *
  * @author Gerhard Riegler - Initial contribution
  */
 public class AstroHandlerFactory extends BaseThingHandlerFactory {
-    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.union(SunHandler.SUPPORTED_THING_TYPES,
```

Comment:
Here I would prefer to keep the Guava usage as it is better than re-implementing the feature in this binding.
Keeping it will allow us to find such places and possibly provide our own implementation in the core framework in future.